src/data/faq.json

```diff
@@ -491,16 +491,29 @@
-                            "Bluetooth compatibility checking and warnings were introduced with Android Corona-Warn-App version 2.2."
+                            "Since version 2.2, the Corona-Warn-App for Android checks whether your device is fully capable of all needed Bluetooth-related features. If you are seeing this message, the operating system of your device lacks the ability to <i>send</i> random IDs via Bluetooth. However, it is still able to <i>receive</i> random IDs. Therefore, <b>your own risk assessment is not affected</b>.",
+                            "The only limitation which this incompatibility causes is that it is not possible for you to warn contacts discovered via Bluetooth. However, users who were checked in to the same event or the same location, at the same time as you were, can still be warned. None of the other features are affected.",
+                            "To resolve the problem, please ensure that you are using the latest version of your smartphone's operating system. If this doesn't help, please <a href='https://support.google.com/android/answer/3094742?hl=en'>contact your device manufacturer</a>.",
+                            "<b>Both Check-in and Contact Journal functionality, as well as retrieving PCR and rapid test results, are fully functional without restriction</b> and only proximity-based ID recording is affected.",
+                            "<hr/>",
+                            "Technical explanation: Your hardware is capable of Bluetooth Low Energy, but your Bluetooth driver is missing a capability called “peripheral mode”. This can only be fixed by your device manufacturer through an update of your operating system. Because the capability is missing, the Corona-Warn-App cannot cause your device to periodically broadcast its own random IDs via Bluetooth."
+                        ]
+                    },
+                    {
+                        "title": "[Google/Android] An “Incompatibility warning” card on my Start Screen is indicating that my device is not compatible with Exposure Notifications at all. What does this mean?",
+                        "anchor": "full_incompat",
+                        "active": true,
+                        "textblock": [
+                            "This does not usually happen on unmodified end-user devices. Therefore, this section doesn't apply to the vast majority of devices. Most devices are, at most, <a href='#part_incompat' target='_blank' rel='noopener'>partially incompatible</a>.",
+                            "Since version 2.2, the Corona-Warn-App for Android checks the Bluetooth capabilities of your device. If a warning message that links to this FAQ entry is shown, your device is not capable of Bluetooth Low Energy and can therefore neither send nor receive random IDs.",
+                            "Your risk assessment will only be based on events and locations you were checked in to.",

```

Comment:
Having thought about this again, since risk assessment is toggled together with Exposure Logging, you can't actually enable it (on devices without Bluetooth), can you?

Reply:
@fynngodau 
I can confirm on Pixel 3a API 29 emulator (which has no Bluetooth support) that it is **not** possible to activate exposure logging. Attempting to activate produces a Cause: 3 error. Clicking on the DETAILS button in the emulator produces no detailed error output.

This probably needs to be raised as an issue in https://github.com/corona-warn-app/cwa-app-android, because the text output "You can send and receive warnings that result from check-ins." by the app is inaccurate. I will leave that to you unless you say you want me to do something. Hopefully there will be very few users affected by this.

I don't have any physical Android device without Bluetooth to test on. The emulators cannot always be relied on to show the exact same behavior that a physical device shows, so please take the results of my emulator test with caution.

Reply:
I am seeing some entries in the log file such as
`2021-05-28T05:38:30.598Z  V/TracingRepository: TracingProgress: TracingProgress.IsCalculating, isExposureDetecting=false, isEWDownloading=false, isEWCalculatingRisk=true, isPTDownloading=false, isPTCalculatingRisk=false`
EW = Exposure Window
PT = Presence Tracing
I was not checked into any events in my test though.
Maybe it does work without activating exposure logging?